docs/guides/reward-distributor-bot.md

```diff
@@ -0,0 +1,64 @@
+# Reward distribution bot
+
+## Introduction
+
+Permissionless reward distribution bot for Lido staking modules. Operates with smart contract based on the [Node Operator Registry](/contracts/node-operators-registry) smart contract. 
+After the [Accounting Oracle](https://docs.lido.fi/guides/oracle-spec/accounting-oracle) completes the third phase, anyone can initiate reward distribution to allocate rewards among Node Operators in the Staking Module, unless the oracle sends the next frame report.
+
+## Requirements
+
+### Hardware
+
+- 1-core CPU
+- 2GB RAM
+
+### Nodes
+
+- Execution Node
+
+## How to use
+
+Every epoch daemon checks the staking modules provided in environment. If a module has a non-distributed rewards, the bot pulls the trigger and distributes rewards between Node Operators inside this module.
+
+Basically bot is watching [RewardDistributionState](https://docs.lido.fi/contracts/node-operators-registry#getrewarddistributionstate) of module. 
```

Comment:
```suggestion
Basically bot is watching [RewardDistributionState](/contracts/node-operators-registry#getrewarddistributionstate) of module. 
```

why it is important: internal links are verified upon building stage and broken links are visible in CI